docs/source/use_CA.md

```diff
@@ -0,0 +1,205 @@
+
+For more information about the folders and certificates that are needed in both an org MSP (also known as a "channel MSP", since it is added to a channel) and the local MSP of a node, check out [MSP structure](https://hyperledger-fabric.readthedocs.io/en/master/membership/membership.html#msp-structure).
+
+### Create the org MSP needed to add an org to a channel
+
+The organizations in a Fabric network don't exist in a physical sense the way nodes do. Rather, they exist as a structure of folders and certificates on the configuration of a channel. These certificates identify the relevant root CA, intermediate CA (if one was used), TLS CA, and at least one admin identity. As you recall from the membership topic and the registration and enrollment steps above, these folders and certificates are returned by the Fabric CA client when enrolling an admin identity, which is why the act of enrolling an admin and the act of "creating an organization" are closely related.
+
+Here is a sample of the folder structure you will need if you want to add an organization to a channel (the structure might vary slightly depending on the method you use to add an organization to a channel, but whatever the method, these are the files and folders you will need):
+
+```
+<location of msp>/msp
+├── cacerts
+│   └── cacert.crt
+├── intermediatecerts
+|   └── cacert.crt
+├── tlscacerts
+│   └── tlsca.<org-domain>.pem
+└── tlsintermediatecerts
+    └── tlsca.<org-domain>.pem
+├── users
+    └── <admin public cert>.pem
+```
+
+Where the folders and certificates are:
+
+* `cacerts`: the root cert of the organization CA where the identity of the admin was registered and enrolled.
+* `intermediatecerts`: the root cert of an intermediate CA, if one was used.
+* `tlscacerts`: the root cert of the TLS CA that has issued certificates to the nodes associated with this organization.
+* `tlsintermediatecerts`: the root cert of the intermediate TLS CA, if one was used.
+* `users`: this folder must contain the **public** certificate of at least one admin of this organization. Do not add a private certificate here. If you have registered and enrolled multiple admins, you will only need to include the public certificate from those enrollments here. You will still need to point to the relevant private certificate to sign requests, however.
+
+Note that while the certificates themselves can be named anything you want, you should not change the name of the folders themselves, as Fabric expects to consume folders with certain names.
+
```

Comment:
As a suggested change, consider adding a Pro Tip:
"In certificate naming, it is helpful to use a convention that will help you keep visual track of whether you are referencing a public certificate or a private key.  Both have the .pem extension, so consider the following convention for naming public certs and private keys:

Rename a public cert from cert.pem to something meaningful like org1-admin-cert.pem
Rename a private key from 94u498f9r9fr98t49t345545345_sk to something meaningful like org1-admin-key.pem

Reply:
This is a really good suggestion.